src/data/finsupp/basic.lean

```diff
@@ -2352,10 +2359,6 @@ instance [monoid R] [monoid S] [add_monoid M] [distrib_mul_action R M] [distrib_
   smul_comm_class R S (α →₀ M) :=
 { smul_comm := λ r s a, ext $ λ _, smul_comm _ _ _ }
 
-instance [monoid R] [add_monoid M] [distrib_mul_action R M] [distrib_mul_action Rᵐᵒᵖ M]
```

Comment:
I think this change should just be reverted, your proof is longer than the one in master it replaced, and I see lily point in moving it.

Reply:
Yes, it's just a remainder of some other changes which I have since reverted.